addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxDevicesHandler.java

```diff
@@ -579,39 +578,6 @@ private void setDeviceConfiguration(Device device) {
         }
     }
 
-    /* (non-Javadoc)
-     * @see org.eclipse.smarthome.core.thing.binding.BaseThingHandler#bridgeStatusChanged(org.eclipse.smarthome.core.thing.ThingStatusInfo)
-     */
-    @Override
-    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
-        logger.debug("Bridge Status updated to {} for device: {}", bridgeStatusInfo.getStatus().toString(),
-                getThing().getUID().toString());
-        getMaxCubeBridgeHandler();
-        if (bridgeStatusInfo.getStatus().equals(ThingStatus.ONLINE)) {
-            // No action
-        } else {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);
-            forceRefresh = true;
```

Comment:
hi @kaikreuzer 
I assumed bridgeStatusChanged was not depreciated.  Is there a specific reason this was removed?

I guess now  `updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE);` is no longer needed, but the `forceRefresh = true;` still is desired for the binding.